specification/iotcentral/resource-manager/Microsoft.IoTCentral/stable/2018-09-01/iotcentral.json

```diff
@@ -537,22 +583,37 @@
     },
     "ErrorDetails": {
       "description": "Error details.",
+      "properties": {
+        "error": {
+          "$ref": "#/definitions/ErrorResponseBody"
```

Comment:
Please add `x-ms-flatten: true` to reduce breaking change from the client side

Reply:
See https://github.com/Azure/autorest/blob/master/docs/extensions/readme.md#x-ms-client-flatten for documentation.

Reply:
xxsx